test/Concurrency/issue-80363.swift

```diff
@@ -0,0 +1,47 @@
+// RUN: %target-typecheck-verify-swift
+
+// https://github.com/swiftlang/swift/issues/80363
+
+class C {}
+
+func testLocal() {
+  isolated let c = C()
+  // expected-error@-1 {{'isolated' may only be used on 'deinit' declarations}}
```

Comment:
the error message isn't quite right. it may also be used on parameters today: `func p(a: isolated A)` where `A` is an `: Actor`